.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Built image '...'
+2. Ran command '....'
+3. See error
+
+**Logs**
+Add logs to help explain your problem
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Specs (please complete the following information):**
```

Comment:
Thought: I am happy to remove the "(please complete ... information)" message. I'm guessing you want to indicate that this section is **required**, but then the other sections seem like they're optional. 😅

Reply:
sounds good, i can take it out! 😄